KishonStClair.js.txt

```diff
@@ -0,0 +1,90 @@
+Get Even
+//1
+for ( i=2 ; i<201 ; i+=2 ) {
+    console.log(i); 
+}
+
+Question 2
+//
+for (var i= 0; i <= 10; i++) {
+    console.log("Love me, pet me! HSSSSSS!"); 
+}
+
+
+
+
+
+
+
+
+
+
+
+
+THERMOSTAT
+
+let currentTemp = Math.floor(Math.random() * 100);
+let desiredTemp = 69;
+ if(currentTemp <= !25 && desiredTemp > 68){
+ console.log("The temperature is " + currentTemp);
+// condition for the temperature under 45f
+} else if(currentTemp < !45 +25 && desiredTemp > 68){
+  console.log("The current temperature is now " + currentTemp);
+// condition for the temperature over 75f
+} else if(currentTemp > !75 -15 && desiredTemp > 68) {
+  console.log("Its a little hot in here " + currentTemp);
+}
+
+
+
+
+
+Fizz Buzz
+
+for (var i = 1; <= 100 i++){
+    if(i % 15 === 0) console.log("FizzBuzz");
+    else if (i % 3 === 0) console.log("Fizz");
+    else if (i % 5 === 0) console.log("Buzz");
+    else console.log(i)
+}
```

Comment:
this works, but it's important to use those curly brackets after your `if (condition)` and `else if (condition)`. 

also, when I ran this, I got an error in line 44. see if you can figure out what it is!